### DISCUSSION TITLE
Multiple SES credentials possible?

### DISCUSSION THREAD

**Discussion by Canos** [awspring/spring-cloud-aws#913] [General]
Hi people, we are trying to use the same application to send emails from two different amazon identities IAM.
I haven't noticed this is available in the docs.

Since the AwsCredentials Interface doesnt have a List, we think our idea is simple not possible.

```java
public interface AwsCredentialsProvider {
    AwsCredentials resolveCredentials();
}
```

can you confirm or giving me an example.
Thanks in advance

**Comment by maciejwalkowiak** [comment]
It should be possible but you would need to opt-out from auto-configuration and put beans together manually.

**Reply by Canos** [reply]
I haven't seen this in the doc. 
If you consider this can help people adding an example in the docs can be great.
Going out of auto-config is not possible to me, so I solved deploying multiple apps with different profiles instead of just one with multiple configs. It suits our infra and our needs.
Thanks for your time
